### PR TITLE
beta version

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
- [DEMO LINK](https://Anton-Zhytonbaiev.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Anton-Zhytonbaiev.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-fri calendar--length-30">
+      <div class="calendar__day"></div>
```
Comment (Anastasiia-Khudych) [inline src/index.html:11]: Modifiers should be written with two dashes by BEM. Like` calendar--start-day--fri`

File: src/styles/blocks.scss/calendar__day.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  }
+}
+
+@for $index from 1 through 31 {
+  .calendar__day:nth-child(#{$index})::before {
+    content: "#{$index}";
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+}
```
Comment (Anastasiia-Khudych) [inline src/styles/blocks.scss/calendar__day.scss]: `Calendar__day` is not a block, so you shouldn't place it in a separate file.

File: src/styles/blocks.scss/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  .calendar--start-#{$day} .calendar__day:first-child {
+    margin-left: $margin-left;
+  }
+}
+
+@for $date from 28 through 31 {
+  .calendar--length-#{$date} .calendar__day:nth-child(n + #{$date + 1}) {
+    display: none;
+  }
+}
```
Comment (Anastasiia-Khudych) [inline src/styles/blocks.scss/calendar.scss]: You could write it using nesting, in the `calendar` selector. The same situation with the` calendar__day` and `for `loop.

**Review by Anastasiia-Khudych — CHANGES_REQUESTED** [review]
Good job! But the code needs some changes.

File: src/index.html (hunk 1 above)
Comment (denys-cheporniuk) [reply src/index.html:11]: `calendar--start-day--fri` isn't a correct modifier, with this modifier you created modifier `--fri` from `--start-day` modifier, according to bem you can't create modifier from another modifier

File: src/styles/blocks.scss/calendar__day.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  &:hover {
+    cursor: pointer;
+    background-color: pink;
+  }
+
+  @for $index from 1 through 31 {
+    &:nth-child(#{$index})::before {
+      content: "#{$index}";
+      font-family: Arial, sans-serif;
+      font-size: 30px;
```
Comment (denys-cheporniuk) [inline src/styles/blocks.scss/calendar__day.scss]: these code shouldn't be in the loop

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]
Screenshot 2022-12-15 at 18 25 53

 calendar should be in the center of the page

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**